api/oneclick.py

```diff
@@ -369,6 +392,13 @@ def update_availability(self, licensepool):
         pass
 
 
+    def internal_format(self, delivery_mechanism):
+        """We don't need to do any mapping between delivery mechanisms and
+        internal formats, because each title is only available in one
+        format.
+        """
+        return delivery_mechanism
+    
```

Comment:
The superclass implementation of this method, which is always called, assumes the existence of a mapping that isn't necessary for OneClick.